res/values/strings.xml

```diff
@@ -610,6 +625,7 @@
     <string name="contact_request_title_desktop">Contact request</string>
     <string name="delete_message_desktop">Delete message</string>
     <string name="more_info_desktop">More info</string>
+    <string name="switch_account_desktop">Switch account</string>
```

Comment:
we should not add the _desktop suffix where it is not really desktop-specific.

so, the _desktop suffix should go only to strings that exist eg. on android and desktop, but differ in details.

i would not see this as a blocker, however, maybe it makes sense to adapt the desktop first.

Reply:
Desktop developers did this. All I'm doing is just add the ones in the untranslated_en.json file.